### PR TITLE
feat: add Float64Map

### PR DESCRIPTION
**Pull request by zuoshuwen**
add Float64Map()

### CODE REVIEW

**Review by stevenh — APPROVED** [review]

**Comment by zuoshuwen** [comment]
Hey @stevenh , thanks for your advice

File: redis/reply.go [hunk 1]
```diff
@@ -476,6 +476,36 @@ func Int64Map(result interface{}, err error) (map[string]int64, error) {
 	return m, nil
 }
 
+// Float64Map is a helper that converts an array of strings (alternating key, value)
+// into a map[string]Float64. The HGETALL commands return replies in this format.
```
Comment (stevenh) [inline redis/reply.go:480]:
```suggestion
// into a map[string]float64. The HGETALL commands return replies in this format.
```

**Review by stevenh — CHANGES_REQUESTED** [review]
Sorry I missed a typo

**Comment by zuoshuwen** [comment]
Thanks for taking this PR :)

**Comment by stevenh** [comment]
Thank you for the PR 🥇

**Pull request merged**